x-pack/plugins/serverless_security/public/components/get_started/sections.tsx

```diff
@@ -85,7 +85,7 @@ export const sections: Section[] = [
       {
         icon: { type: protect, size: 'xl' },
         title: i18n.PROTECT_YOUR_ENVIRONMENT_TITLE,
-        id: GetSetUpCardId.protectYourEnvironmentInRuntime,
+        id: GetSetUpCardId.protectYourEnvironmentInRealtime,
         productTypeRequired: [
           ...ActiveConditions.cloudToggled,
           ...ActiveConditions.endpointToggled,
```

Comment:
Why don't we just do:
```suggestion
        productTypeRequired: [
          ProductLevel.cloud,
          ProductLevel.endpoint,
```

The `ActiveConditions` can be removed

Reply:
The naming Yuliia defined from the Control Plane config is:
- productLevel: `'security' | 'endpoint' | 'cloud'`
- productTier: `'essentials' | 'complete'`
- productType: the combination of both: `{productLevel, productTier }`

We are currently using _productLevel_ entries in the `productTypeRequired` attr, can we do a quick search/replace `productTypeRequired` -> `productLevelRequired`, to be consistent?